fusion-cli/entries/client-public-path.js

```diff
@@ -4,7 +4,8 @@ declare var __webpack_public_path__: string;
 declare var __webpack_nonce__: string;
 */
 
-import '@babel/polyfill';
+import 'core-js/stable';
```

Comment:
Does this still work with the env preset `useBuiltIns: 'entry'` option? I recall this was hardcoded to `@babel/polyfill` for while.

Reply:
It looks like `useBuiltins: 'entry'` replaces `@babel/polyfill` with the imports directly from `core-js` (https://babeljs.io/docs/en/babel-preset-env#usebuiltins-entry). I'm not entirely sure why the notices told me to replace the import for polyfill since it seems to do it automatically